packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -1,14 +1,24 @@
 import { BigNumber } from 'bignumber.js'
+import { SingletonRouter as Router, withRouter } from 'next/router'
 import * as React from 'react'
 import { Text as RNText, View } from 'react-native'
 import ValidatorsListRow, { CeloGroup } from 'src/dev/ValidatorsListRow'
 import { styles } from 'src/dev/ValidatorsListStyles'
 import { H1 } from 'src/fonts/Fonts'
 import { I18nProps, withNamespaces } from 'src/i18n'
 import Chevron, { Direction } from 'src/icons/chevron'
+import Button, { BTN } from 'src/shared/Button.3'
+import menu from 'src/shared/menu-items'
+import OvalCoin from 'src/shared/OvalCoin'
 import { colors, standardStyles, textStyles } from 'src/styles'
 import { weiToDecimal } from 'src/utils/utils'
 
+const networkMenu = [
+  // ['Mainnet', menu.VALIDATORS_LIST.link],
+  ['Baklava', menu.VALIDATORS_LIST__BAKLAVA.link],
+  ['Baklavastaging', menu.VALIDATORS_LIST_BAKLAVASTAGING.link],
```

Comment:
I believe we will want alfajores here too based on what @timmoreton said

Reply:
Yes, I'm waiting for the rest of the information like mainnet to finish this part. Right now is more for feedback purposes.